addons/stock/models/stock_move.py

```diff
@@ -1080,9 +1080,10 @@ def _prepare_move_line_vals(self, quantity=None, reserved_quant=None):
             'picking_id': self.picking_id.id,
         }
         if quantity:
+            rounding = self.env['decimal.precision'].precision_get('Product Unit of Measure')
```

Comment:
The name `rounding` is a bit confusing to me, as it sounds more like the rounding method.  I suggest to use `precision` or `precision_digits` instead.